docs/rules/no-js-extension.md

```diff
@@ -2,8 +2,15 @@
 
 ## Rule Details
 
-Dependency paths in RequireJS should not use a `.js` extension, since the path could be a for a package directory.
+Dependency paths in RequireJS should not use a `.js` extension, since the path could be a for a package directory. Extension can be used only in a combination with a plugin prefix.
```

Comment:
Grammar thing: "Extensions may only be used in combination with a plugin prefix."